exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetFormatConfig.java

```diff
@@ -24,9 +24,16 @@
 import com.fasterxml.jackson.annotation.JsonTypeName;
 
 @JsonTypeName("parquet") @JsonInclude(JsonInclude.Include.NON_DEFAULT)
-public class ParquetFormatConfig implements FormatPluginConfig{
+public class ParquetFormatConfig implements FormatPluginConfig {
 
-  public boolean autoCorrectCorruptDates = true;
+  private boolean autoCorrectCorruptDates = true;
```

Comment:
4 unit test have failed in `TestCorruptParquetDateCorrection` class.
For Json generation let's leave `autoCorrectCorruptDates` as public and `areCorruptDatesAutoCorrected()` method mark as `@JsonIgnore`.